src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -113,12 +114,27 @@ private string BuildModuleKey(string projectGuid)
             await SynchronizeSuppressedIssues();
         }
 
+        private Task DoInitialFetch()
+        {
+            // We might not have connected to the server at this point so if necessary
+            // wait before trying to fetch the issues
+            int retryCount = 0;
+            while (!this.sonarQubeService.IsConnected && retryCount < 30)
+            {
+                Thread.Sleep(1000);
+                retryCount++;
+            }
+            return SynchronizeSuppressedIssues();
+        }
+
         private async Task SynchronizeSuppressedIssues()
         {
             try
             {
                 if (!this.sonarQubeService.IsConnected)
                 {
+                    // TODO: log to output window
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)